startup.d/20-syntax-check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python -m py_compile $(find . -name '*.py');
+if [ "$?" != "0" ]
+then
+  echo "There is syntax error. This may endanger chaos. Pauses updates."
+  killall bash
```

Comment:
No, chaos will not be stopped. Chaos is not run by bash and through the supervisord. If the update process is interrupted, it will not affect the server, so server can merge a new PR which fix errors.